### DISCUSSION TITLE
Are the pre and post launch script env vars available inside the launched instance?

### DISCUSSION THREAD

**Discussion by mainrs** [PrismLauncher/PrismLauncher#2754] [Q&A] [answered]
Id like to know what instance a user launched inside my mod. Is that possible somehow? 

I thought this might be more reliable than checking the absolute path of a class inside my mod jar. And then seducing it's location based on the returned file path.

**Comment by TheKodeToad** [comment, marked as answer]
You can use
```java
String instanceName = System.getProperty("org.prismlauncher.instance.name");
```
See https://github.com/PrismLauncher/PrismLauncher/blob/develop/libraries/launcher/org/prismlauncher/SystemProperties.java to see all of the properties that are available.
